Makefile

```diff
@@ -154,6 +154,12 @@ $(TEST_DIR)/issue_65/optimized/issue_65.go: $(TEST_DIR)/issue_65/issue_65.peg $(
 $(TEST_DIR)/issue_65/optimized-grammar/issue_65.go: $(TEST_DIR)/issue_65/issue_65.peg $(BINDIR)/pigeon
 	$(BINDIR)/pigeon -nolint -optimize-grammar $< > $@
 
+$(TEST_DIR)/issue_70/issue_70.go: $(TEST_DIR)/issue_70/issue_70.peg $(BINDIR)/pigeon
```

Comment:
Why have you removed the additional lines for the optimized parsers? (see [here](https://github.com/mna/pigeon/blob/0cb2d5b52ca30953f163b431db5ffba5101cffe6/Makefile#L160-L164))?

Reply:
Probably a mistake when fixing the makefile conflict. Added them back.